### DISCUSSION TITLE
Should we rename the SablierV2Batch contract?

### DISCUSSION THREAD

**Discussion by andreivladbrg** [sablier-labs/v2-periphery#318] [Ideas]
With the new OE contract, we will have multiple batch contracts, but in different repos.

Thus, to differentiate between them, shouldn't we rename the batch contract to:` SablierV2LockupBatch`?

@sablier-labs/solidity 

Also, tagging @razgraf to ask if this would improve the dev work at the app level?

**Comment by PaulRBerg** [comment]
I like this idea.

**Reply by razgraf** [reply]
No necessary improvements in dev work, but separations of concerns is important so I'm pro renaming this to LockupBatch 👍

**Comment by smol-ninja** [comment]
+1 to this.

**Comment by PaulRBerg** [comment]
Great. Let's rename to `BatchLockup` - clearer than `LockupBatch` (which could be misunderstood for a new distribution model).
